### DISCUSSION TITLE
New PC, old plugs not seen

### DISCUSSION THREAD

**Discussion by Raynorshyn** [apohl79/audiogridder#1300] [General]
I just built an upgrade PC, the original PC is in a different location. I've reloaded all the Reaper projects, VSTs and AG. A multitrack project wasn't able to load the audiogridder plugins made with the other PC build. Is it because it is considered a different server that it connects to? Is there a way to trick it? Or, do I need to go to the original PC and make AG presets and bring them over? Or, worse yet, take pictures of the plugin settings and remake them all on the new PC. = alotta work.

Any ideas?

**Comment by apohl79** [comment]
You can reconnect all plugins at once via the tray icon.

**Comment by Raynorshyn** [comment]
Andreas,

I don't understand, this is what each looks like in the Reaper FX rack:
[image: AG issue.png]

Thanks, Tom

**Reply by Raynorshyn** [reply]
When opening a Reaper project that includes plugins no longer installed, it gives you splash screen, listing the plugins and on what tracks. It says it will keep the configuration for next time.

On the track the FX is named there, but no content. 

The Audio Gridder tray, top right, is not loaded with anything as the project didn't load the plugins.
All of the content, vsts, that would be inside the instance of AG are available on the PC. AG just doesn't load itself or the contents.
Looks like this:
 
<img width="560" alt="AG issue" src="https://github.com/apohl79/audiogridder/assets/76830139/eeb24848-1058-4538-a764-e12da8b8cbe4">
<img width="560" alt="AG issue" src="https://github.com/apohl79/audiogridder/assets/76830139/d0f0c17b-74f8-4909-81bd-a0759315d800">

Any other help?
